packages/app-staking/src/Account/index.tsx

```diff
@@ -58,12 +60,28 @@ class Account extends React.PureComponent<Props, State> {
     isValidatingOpen: false,
     isUnbondOpen: false,
     bondedId: null,
+    recentlyOffline: {},
     sessionId: null,
     stashId: null
   };
 
-  static getDerivedStateFromProps ({ session_nextKeyFor, staking_bonded, staking_ledger }: Props, state: State): Partial<State> {
+  static getDerivedStateFromProps ({ session_nextKeyFor, staking_bonded, staking_ledger, staking_recentlyOffline = [] }: Props, state: State): Partial<State> {
     return {
+      recentlyOffline: staking_recentlyOffline.reduce(
```

Comment:
Can we not rather calculate this on the main index and pass it in? (So instead of each account having a sub, which is cached, and doing the same thing)